en/companion_computer/auterion_skynode.md

```diff
@@ -0,0 +1,66 @@
+# Auterion Skynode
+
+[Skynode](https://auterion.com/product/skynode/) is a powerful flight computer that combines a mission computer, flight controller, video streaming, networking, and cellular connection, in a single tightly integrated device.
+
+![Auterion Skynode (Enterprise)](../../assets/companion_computer/auterion_skynode/skynode_enterprise_tdw.png)
+
+The onboard software is Auterion OS, consisting of an enterprise-hardened version of PX4 running on the flight controller, and an operating system with advanced management software running on the mission computer.
+The OS is managed by Auterion in production, with customer applications running as "add ons" in a safe sandbox within the mission computer.
+
+Auterion OS and Skynode allow seamless integration with Auterion's other software and fleet management products.
+
+For information about Auterion and Skynode, including purchase links:
+
+- [auterion.com](https://auterion.com/)
+- [Skynode](https://auterion.com/product/skynode/) (auterion.com)
+- Skynode Guides:
+  - [Manufacturer's Guide](https://docs.auterion.com/manufacturers/getting-started/readme) 
+  - [App Developer's Guide](https://docs.auterion.com/developers/getting-started/readme)
+
+
+## Skynode with Vanilla PX4
+
+Skynode comes with a managed release of PX4.
+If you would like to try a more recent PX4 flight kernel, you can install the upstream "vanilla" PX4 from [PX4/PX4-Autopilot](https://github.com/PX4/PX4-Autopilot).
+
+Upstream PX4 will generally work, with the following caveats:
+
+- Configuration for your exact product may not be present.
```

Comment:
```suggestion
- No officially supported release.
- Configuration for your exact product may not be present.
```
I think that's important to mention.

Reply:
This is strongly implied by the "you can try this out on your development image". 

But if we must, should be at the end given the context. Proposed alternative text in https://github.com/PX4/PX4-user_guide/pull/2631/files#r1267464125

Reply:
I don't insist on that point but the reality is that not any random version flashed is/can be supported. So I thought it might be worth mentioning. You're right that it was already implied.

Reply:
Ah, you mean "PX4 does not support this frame"/This is a manufacturer supported frame. Let me add that to the note below.